Earthfile

```diff
@@ -230,6 +230,9 @@ docker:
 
     # Copy kairos binaries
     COPY +build-kairos-agent/kairos-agent /usr/bin/kairos-agent
+
```

Comment:
mmm I wonder how this can work. the webui-deps end up to be in the docker image, not in the source code that is going to be compiled. The target that builds the agent is `build-kairos-agent`.

When running `earthly +build-kairos-agent` I expect an agent binary with the webui assets included inside (also because otherwise the binary would not container the assets, and the webui would be broken)

Reply:
good catch, it was only working because I didn't clean my local node_modules directory :man_facepalming: ... I'll look into it

Reply:
ok, it should be working properly now